File: app/src/main/java/com/katdmy/android/unocount/Player.java
package com.katdmy.android.unocount;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Player {
    @PrimaryKey
    @NonNull
    private String mName;

    private boolean mActive;

    Player(@NonNull String name) {
        mName = name;
        mActive = true;
    }

    public String getName() {
        return mName;
    }

    boolean getActive() {
        return mActive;
    }

    void setActive(boolean active) {
        mActive = active;
    }
}

File: app/src/main/java/com/katdmy/android/unocount/domain/player/model/Player.kt
package com.katdmy.android.unocount.domain.player.model

import android.os.Parcelable
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.Index
import androidx.room.PrimaryKey
import com.katdmy.android.unocount.data.db.UnoCountContract
import kotlinx.parcelize.Parcelize

@Parcelize
@Entity(
        tableName = UnoCountContract.PlayerEntry.TABLE_NAME,
        indices = [Index(UnoCountContract.PlayerEntry.COLUMN_NAME)]
)
data class Player(
        @PrimaryKey
        @ColumnInfo(name = UnoCountContract.PlayerEntry.COLUMN_NAME)
        val name: String = "",
        @ColumnInfo(name = UnoCountContract.PlayerEntry.COLUMN_ACTIVE)
        var active: Boolean = true
) : Parcelable